src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -0,0 +1,153 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageTags>aspnetcore</PackageTags>
+    <Description>Microsoft.AspNetCore.All</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- set PrivateAssets=None to ensure that all assets, including Build and Analyzer, are included in the nuspec -->
+    <PackageReference Include="Microsoft.AspNetCore.Antiforgery" Version="$(AspNetCoreVersion)" PrivateAssets="None" />
+    <!--<PackageReference Include="Microsoft.AspNetCore.ApplicationInsightsLightup" Version="$(AspNetCoreVersion)" PrivateAssets="None" />-->
```

Comment:
Actually these shouldn't be removed. A few of the packages need to be in the cache but not the metapackage and the others need to be included but are blocked based on other issues. I'll add some notes linking to the individual issues.